extensions/csv/src/main/java/io/deephaven/csv/parsers/DateTimeAsLongParser.java

```diff
@@ -0,0 +1,56 @@
+package io.deephaven.csv.parsers;
+
+import io.deephaven.csv.sinks.Sink;
+import io.deephaven.csv.tokenization.Tokenizer;
+import io.deephaven.csv.parsers.context.ParseContext;
+
+public final class DateTimeAsLongParser extends ParserBase<long[]> {
+    public static final DateTimeAsLongParser INSTANCE = new DateTimeAsLongParser();
+
+    private DateTimeAsLongParser() {}
+
+    @Override
+    public Sink<long[]> tryParse(ParseContext ctx, IteratorHolder ih, IteratorHolder ihAlt) {
+        return twoPhaseDriver(ctx, ih, ihAlt, DateTimeAsLongParser::tryParseHelper,
+                ctx.sinkFactory::makeDateTimeAsLongSink);
+    }
+
+    private static boolean tryParseHelper(ParseContext ctx, IteratorHolder ih,
+                                          Sink<long[]> sink, long current, long end) {
+        final long[] chunk = new long[DEST_BLOCK_SIZE];
+        final long[] dateTimeAsLongHolder = new long[1];
```

Comment:
I think we prefer MutableLong for this sort of thing, instead of an array of size 1.  Search (find + grep) for MutableLong on any engine files to see which one to pick up, if you just let IDEA find one for you there are too many choices.

Reply:
That's great. I had mistakenly thought the MutableXXX classes were not linked to our project, but that was just a mistake on my part.